### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by davidjmurphyjr**
A few small updates to the docs that I thought might help with future onboarding.

### CODE REVIEW

File: apps/api/.env.example [hunk 1]
```diff
@@ -13,7 +13,10 @@ ALGOD_PORT=
 ALGOD_TOKEN=
 ALGOD_ENV=testnet
 
-# CMS configuration (CMS_ACCESS_TOKEN should match the KEY environment variable in the CMS .env)
+### CMS configuration
+# CMS_ACCESS_TOKEN should match...
+# - the KEY environment variable in the CMS .env
+# - the Admin User > Admin Options > Token field in the CMS UI
```
Comment (smonn) [inline apps/api/.env.example:19]: Sorry, this is incorrect (the old one was wrong too). This should **not** match the `KEY` environment variable from the CMS. That is just used for the CMS session cookie I believe. `CMS_ACCESS_TOKEN` should **only** match the latter (Admin User > Admin Options > Token field in the CMS UI).

File: apps/api/.env.example [hunk 2]
```diff
@@ -59,6 +62,7 @@ CURRENCY=
 COINBASE_URL=https://api.coinbase.com
 
 # pinata.cloud configuration (for storing on IPFS)
+# sign up for a free tier pinata.cloud account using your Rocket email
```
Comment (smonn) [inline apps/api/.env.example:65]: Not Rocket 😉 

```suggestion
# You can sign up for a free tier https://pinata.cloud account with 1 GB storage to start
```

**Review by smonn — COMMENTED** [review]
Thanks! Some small tweaks are needed, but looks good.

File: apps/api/.env.example (hunk 1 above)
Comment (davidjmurphyjr) [reply apps/api/.env.example]: This process was documented in the some of the main and CMS READMEs. I moved it to the API README, as that seemed like the right place for it.

**Review by smonn — APPROVED** [review]
Thanks for all of this, LGTM!

**Pull request merged**